CHANGELOG.md

```diff
@@ -1,24 +1,5 @@
 # Changelog
 
-### 2.19.0
-This release includes:
-* An Amazon Linux 2 Base
-* Fluent Bit [1.8.3](https://fluentbit.io/announcements/v1.8.3/)
-* Amazon CloudWatch Logs for Fluent Bit 1.6.1
-* Amazon Kinesis Streams for Fluent Bit 1.7.1
-* Amazon Kinesis Firehose for Fluent Bit 1.6.0
-
-### 2.18.0
-This release includes:
-* An Amazon Linux 2 Base
-* Fluent Bit [1.8.2](https://fluentbit.io/announcements/v1.8.2/)
-* Amazon CloudWatch Logs for Fluent Bit 1.6.1
-* Amazon Kinesis Streams for Fluent Bit 1.7.1
-* Amazon Kinesis Firehose for Fluent Bit 1.6.0
-
-Compared to `2.17.0` this release adds:
-* Feature - [Multiline Filter](https://docs.fluentbit.io/manual/pipeline/filters/multiline-stacktrace) which helps to concatenate messages that originally belong to one context but were split across multiple records or log lines.
-
```

Comment:
don't change the changelog. it is only for documentation. It is not used in the release automation.